bin/dev

```diff
@@ -38,7 +38,8 @@ function _get_local_ip() {
         (>&2 echo -n "==> ${ip} ... ")
 
         # Ping google from this interface to check internet access
-        if ! ping -I "${ip}" -W 1 -c 1 www.google.com &> /dev/null; then
+        ping -I "${ip}" -W 1 -c 1 www.google.com &> /dev/null
+        if [ $? -eq 0 ]; then
```

Comment:
This is not recommended, see https://github.com/koalaman/shellcheck/wiki/SC2181

Reply:
Good to know, thanks !
I'll fix the issue keeping this syntax.